### DISCUSSION TITLE
So I want to use MMM-calenderext2

### DISCUSSION THREAD

**Discussion by Dragontamer333** [edward-shen/MMM-pages#60] [Q&A] [answered]
So i want to use calender ext 2 with the pages module. Can someone point me in the correct direction on how to set it up. When I use it like other modules nothing shows up.

**Comment by edward-shen** [comment, marked as answer]
See the doc in MMM-calendarext2: https://github.com/MMM-CalendarExt2/MMM-CalendarExt2/blob/master/docs/Tip:-Sync-with-MMM-Pages..md

**Reply by Dragontamer333** [reply]
Just want a quick clarification, I don't want to have this rotate through pages automatically. I see some stuff like rotation time, but am not 100 percent sure how it works. Would me basically using this example code allow for the calendar to not rotate, but still change with things like alexa control? Thank you for the quick reply.

**Reply by edward-shen** [reply]
If you don't set any rotation time then MMM-pages won't rotate. If you want to use alexa to control this you must use a module that interacts with it—that's out of my control and knowledge.

**Reply by Dragontamer333** [reply]
Ok thank you for at least pointing me in the right direction. Great module by the way.
